docs/cloud-native-security/cspm-get-started-aws.asciidoc

```diff
@@ -59,21 +59,123 @@ image::images/cspm-cloudformation-template.png[The Add permissions screen in AWS
 
 When you return to {kib}, click *View assets* to review the data being collected by your new integration.
 
+
+[discrete]
+[[cspm-setup-organization-manual]]
+=== Manual authentication for organization-level onboarding
+
+NOTE: If you're onboarding a single account instead of an organization, skip this section.
+
+When using manual authentication to onboard at the organization level, you need to configure the necessary permissions using the AWS console for the organization where you want to deploy:
+
+* In the organization's management account (root account), create an IAM role called `cloudbeat-root` (the name is important). The role needs several policies:
+
+** The following inline policy:
+
+.Click to expand policy
```

Comment:
On the website, the "click to expand" arrow indentation looks off, is there a way to fix it?